src/pages/admin/apply/[id]/[applicantId]/index.tsx

```
@@ -0,0 +1,158 @@
+import { useUpdateApplicantStatus } from '@/hooks/api/apply/useUpdateApplicantStatus';
+import { ApplicantStatus } from '@/types/apply';
+import { getNextStatus } from '@/utils/filter';
+
+type Props = {
+  id: number;
+  applicantId: number;
+};
+
+export default function Index({ id, applicantId }: Props) {
+  const [{ token }] = useCookies(['token']);
+  const { data: data, isLoading } = useApplicantInfo(id, applicantId, token);
```

Biome (1.9.4)

[error] Useless rename.

Safe fix: Remove the renaming.


(lint/complexity/noUselessRename)